exercises/perfect-numbers/.gitignore

```diff
@@ -0,0 +1,10 @@
+# Generated by Cargo
+# will have compiled files and executables
+/target/
+
+# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+# More information here http://doc.crates.io/guide.html#cargotoml-vs-cargolock
+Cargo.lock
+
+# IDE directories
+.idea
```

Comment:
I would think IDE-specific directories should go in a user-specific gitignore, rather than a exercise-specific. I would remove it.

Reply:
removed